organization/_posts/2018-03-15-startup.md

```diff
@@ -0,0 +1,140 @@
+
+News, general matters
+=====================
+-   Research Software Engineers group in the UK contacted us
+    -   Looking for sponsorship for
+        [RSE2018](http://rse.ac.uk/conf2018/) (3-4
+        Sept), which we are not able to give.
+    -   However, participating in their conference looks interesting -
+        anyone from the UK interested?
+-   Are there other conferences where we would like to have an HSF
+    presence this year?
+    -   Purpose has to be clear: Who we are. What we do. Are we/you
+        interested in sharing ideas or resources?
+
+HSF/WLCG Workshop
+=================
+-   ### General points
+    -   199 people registered.
+    -   Vidyo rooms are cloned for all the right sessions.
+-   ### Outcomes
+    -   Discussion this week:
+        -   No template slides yet - early next week (unless someone
+            else volunteers!)
+        -   Oli - HSF curation would be a very important point, let's
+            make that a top item and very visible.
+        -   Ian - WLCG strategy document has some major things that
+            will/should crystallise into working groups
+            -   Hope to circulate the document in advance of Naples
+        -   Note from Dan Katz that
+            [Depsy](http://depsy.org/) project [has
+            ended](https://www.google.com/url?q=https://twitter.com/depsy_org/status/970376969782149120&sa=D&ust=1521057055228000&usg=AFQjCNFH-IBN3HOO2Rp0yGApZBWi6FFAdg).
+-   ### Agenda Updates
+-   #### Monday
+    -   HEP Use Cases
+        -   Presentation by LHCb will go into the agenda now
+        -   Presentation from Simone foreseen - but does this have too
+            much of an overlap with Liz's talk? Need to clarify that.
+-   #### Wednesday
+    -   Programming for Concurrency and Co-Processors
+        -   Lost one talk, so coffee break restored to 30'.
+    -   Visualisation (Parallel)
+        -   Agenda defined.
+        -   Discussion about common projects
+        -   Contributions from:
+            -   CMS/GSI/ROOT
+            -   ATLAS
+            -   Belle II
+    -   Simulation (Parallel)
+        -   Pere will help run the session.
+    -   Technical Discussion on Frameworks (Parallel)
+        -   Session has been further organised. Marco will present one
+            of the topics. Jim K may be able to speak about
+            serialisation.
+-   #### Thursday
+    -   Closing Plenary
+        -   SKA talk is confirmed (Anna Scaife) - we allocated 30'.
+        -   Closing talks on way forward and vision for final 30'.
+        -   Need to populate Indico (Pere and Liz).
+
+CWP
+===
+-   ### General Matters and Roadmap
+    -   CERN Courier article was finalised, the April issue will be live
+        online from 23 Mar and in print a few days after that.
+    -   Symmetry magazine article status. Being chased up internally.
```

Comment:
Apologies for trivial suggestion: one could simply write
"Symmetry magazine article being chased up internally by editor."